backend/app/models/action_log.py

```
@@ -0,0 +1,18 @@
+class ActionLogComment(Base):
+    __tablename__ = "action_log_comments"
+
+    id = Column(Integer, primary_key=True, index=True)
```

Ruff (0.12.2)

Undefined name `Column`

(F821)